html/pfappserver/lib/pfappserver/Form/Config/Authentication/Rule.pm

```diff
@@ -327,6 +328,13 @@ sub validate {
         $self->field('actions')->add_error("You must at least set a role, mark the user as a sponsor, or set an access level.");
     }
 
+    @actions = grep { $_->{type} eq $Actions::SET_ACCESS_LEVEL } @{$self->value->{actions}};
+    if ( scalar @actions > 0 ) {
+        if ( @{$self->value->{actions}} > 1 ) {
+            $self->field('actions')->add_error("'set access level' should be an exclusive action to a rule.");
+        }
+    }
+
```

Comment:
I'm not sure I will add it to the checkup since none of the current constraints in the Form validation are checked by checkup.
Also, we may want... just for the beginning, override that behavior using the flat file.